app/assets/javascripts/fae/_tables.js

```diff
@@ -294,6 +297,10 @@ Fae.tables = {
 
       $fixed_header.append( $header );
       $this.after($fixed_header);
+
+      if ($this.hasClass('tablesorter')) {
+        _this.proxyStickyTableHeaderClicks($fixed_header, $this);
```

Comment:
This function, as well as the newly added `replaceStickyTableHeader` are only invoked in one location each in this file. Are they being called outside this file? I'm pro-modularization, but these both seem like unnecessary public functions. 

I don't think you'd be wrong to include the contents of these functions instead here, since they're only 1 -5 lines. The added `_this` can also be removed.